tests/JSValueTests.Usage.swift

```diff
@@ -135,11 +135,11 @@ class JSValueUsageTests : XCTestCase {
                 "url" : "http://remote.bloxus.com/"
             ]
         
-        let blog = make ⇒
-            (json["id"].number ⇒ toInt) ⇒
-            json["name"].string ⇒
-            json["needspassword"].bool ⇒
-            (json["url"].string ⇒ toURL)
+        let blog = make(
```

Comment:
Yeah, I guess verbosity is the way to go, then. Will add a commit to this PR in a sec.